keyboards/massdrop/thekey/readme.md

```diff
@@ -0,0 +1,20 @@
+# The Key
+
+![The Key](https://massdrop-s3.imgix.net/img_thread/1632776012171.401382764724844340547932-2968copypdp.png?auto=format&fm=jpg&fit=min&w=796&dpr=2&q=35)
+
+The Stack Overflow They Key is ...
+
+Keyboard Maintainer: 
+Hardware Supported: 
+Hardware Availability: 
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Updated to reflect instructions on programming